docs/api/ref/schemas/product_tags.yaml

```diff
@@ -13,17 +13,22 @@ properties:
 
   categories:
     type: string
+    description: |
+      Comma separated list of categories (not taxonomized), in the last language used to edit it (recorded in categories_lc)
+      This field is mostly used for debugging and testing purposes. Do not use it for display purposes.
   categories_hierarchy:
     type: array
+    description: An array of categories tag entries
```

Comment:
```suggestion
    description: |
      An array of categories tag entries.
      
      It only contains categories found in the taxonomy.
```

Reply:
@alexgarel it's not correct, categories_hierarchy contains all categories entries, including those that are not in the taxonomy. The difference with categories_tags is that those entries that are not in the taxonomy are not normalized (e.g. we keep spaces, accents etc.)

Reply:
I'll try to reformulate